packages.adoc

```diff
@@ -109,6 +109,32 @@ license usage. To find out more about {product-short} Archiving and Discovery,
 contact {product-short} sales.
 endif::networkeditiondoc[]
 
+* *{product-short} Chat:* Installing this package is optional. This package should be installed on at least one
+*{product-short}-Store* server. {product-short} Chat includes a server extension with all 
+the necessary modules to run an embedded XMPP Server, and an End-User Zimlet which connects
+to the server extension and offer to the users a rich point-to-point text-chat conversations.
+{product-short} Chat is marked as GA and supported starting {product-name} 8.8.
++
+[NOTE]
+The {product-short} Chat package must be selected and installed on every *{product-short}-Store*.
++
+
+* *{product-short} Drive:* Installing this package is optional.This package should be installed on at least one
+*{product-short}-Store* server. *{product-short} Drive includes a server extension with all 
```

Comment:
Looks like a couple of places here and on the next line where BOLD is started but not finished?